notebooks/validate_and_tokenize_data.ipynb

```diff
@@ -854,7 +902,7 @@
     "n_billing_tokens_in_dataset = len(mds_dataset) * FT_API_args.context_length \n",
     "print(f\"Dataset has ~{n_billing_tokens_in_dataset} tokens that will be charged for during training\")\n",
     "print(f\"By default, you'll train for {n_epochs} epochs on this dataset\")\n",
-    "print(f\"By default, you'll be charged for ~{n_epochs * n_billing_tokens_in_dataset} tokens\")"
+    "print(f\"By default, ~{n_epochs * n_billing_tokens_in_dataset} tokens will be used in training\")"
```

Comment:
instead of ~ use "approximately"
```suggestion
    "print(f\"By default, approximately {n_epochs * n_billing_tokens_in_dataset} tokens will be used in training\")"
```